src/main/java/ludogorie_soft/reservations_platform_api/data/DataLoader.java

```diff
@@ -14,20 +14,20 @@ public DataLoader(RoleRepository roleRepository) {
         this.roleRepository = roleRepository;
     }
 
-    @PostConstruct
-    public void loadRoles() {
-        if (roleRepository.findByName("ADMIN") == null) {
-            Role adminRole = new Role();
-            adminRole.setName("ADMIN");
-            roleRepository.save(adminRole);
-        }
-
-        if (roleRepository.findByName("USER") == null) {
-            Role userRole = new Role();
-            userRole.setName("USER");
-            roleRepository.save(userRole);
-        }
-    }
+//    @PostConstruct
```

Comment:
delete the code instead of commenting it

Reply:
Code deleted